src/main/java/com/box/sdk/BackoffCounter.java

```diff
@@ -50,12 +59,18 @@ public void reset(int maxAttempts) {
         this.attemptsRemaining = maxAttempts;
     }
 
+    /**
+     * Generates some random amount of time to backoff.
+     * Time is within <16000, 48000) ms
+     * @return Time in miliseconds.
+     */
     private int calculateDelay() {
         int exponent = this.maxAttempts - this.attemptsRemaining;
         double minWindow = 1 - RANDOM_FACTOR;
         double maxWindow = 1 + RANDOM_FACTOR;
+        //1.0 + 0.5
         double jitter = (Math.random() * (maxWindow - minWindow)) + minWindow;
-
+        //2^5*1000*1,5
```

Comment:
Not necessarily  2^5 (5 changes) so this comment may be misleading. Add `max 2^5*1000*1,5` or sth like that